src/marqo/core/index_management/index_management.py

```diff
@@ -60,13 +59,22 @@ class IndexManagement:
         '''
     )
 
-    def __init__(self, vespa_client: VespaClient, zookeeper_client: Optional[ZookeeperClient] = None):
+    def __init__(self, vespa_client: VespaClient, zookeeper_client: Optional[ZookeeperClient] = None,
+                 enable_index_operations: bool = False):
+        """Instantiate an IndexManagement object.
+
+        Args:
+            vespa_client: VespaClient object
+            zookeeper_client: ZookeeperClient object
+            enable_index_operations: Flag to enable index operations
```

Comment:
This is vague. Describe what we mean by index operations (what happens if true, or false)

Reply:
done